trio/_threads.py

```diff
@@ -292,11 +292,11 @@ async def to_thread_run_sync(  # type: ignore[misc]
             raise ValueError(
                 "Cannot set `cancellable` and `abandon_on_cancel` simultaneously."
             )
-        warnings.warn(
-            DeprecationWarning(
-                "`cancellable` keyword argument is deprecated, "
-                "use `abandon on cancel` instead."
-            )
+        warn_deprecated(
+            "`cancellable=` keyword argument",
+            "0.23.0",
+            issue=2841,
+            instead="`abandon_on_cancel=` keyword argument",
```

Comment:
```suggestion
            instead="`abandon_on_cancel=`",
```

Reply:
These make the message be:
```
The `cancellable=` keyword argument to `trio.to_thread.run_sync` is deprecated since Trio 0.23.0; use `abandon_on_cancel=` instead (<issue url>)
```
which I think reads nicely.